server/admin/admin_server_test.go

```diff
@@ -146,81 +122,108 @@ func TestServer_ListTrees(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 
+	activeLog := proto.Clone(testonly.LogTree).(*trillian.Tree)
+	frozenLog := proto.Clone(testonly.LogTree).(*trillian.Tree)
+	frozenLog.TreeState = trillian.TreeState_FROZEN
+	deletedLog := proto.Clone(testonly.LogTree).(*trillian.Tree)
+	activeMap := proto.Clone(testonly.MapTree).(*trillian.Tree)
+	deletedMap := proto.Clone(testonly.MapTree).(*trillian.Tree)
+
+	id := int64(17)
+	nowPB := ptypes.TimestampNow()
+	for _, tree := range []*trillian.Tree{activeLog, frozenLog, deletedLog, activeMap, deletedMap} {
+		tree.TreeId = id
+		tree.CreateTime = proto.Clone(nowPB).(*timestamp.Timestamp)
+		tree.UpdateTime = proto.Clone(nowPB).(*timestamp.Timestamp)
+		id++
+		nowPB.Seconds++
+	}
+	for _, tree := range []*trillian.Tree{deletedLog, deletedMap} {
+		tree.Deleted = true
+		tree.UpdateTime = proto.Clone(nowPB).(*timestamp.Timestamp)
```

Comment:
Set `tree.DeleteTime` too for consistency?

Reply:
It should actually be DeleteTime instead of UpdateTime. Fixed.